### PR TITLE
Scroll to created org

### LINKED ISSUE #2457: [Bug] After creating a new organization, please scroll automatically to the new organization
When i create an organization (using Create Organization CTA on home page), the page should scroll to where the new organization lies. Today, I have to scroll on my own. The expectation is that when you create an organization, you most probably want to create applications inside the organization, add new members, etc. For that, the organization should be in focus.

### PR DESCRIPTION
**Pull request by rishabhsaxena**
## Description
Scroll to created org

Fixes #2457

## Type of change
- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?
- manually

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by hetunandu — APPROVED** [review]

**Review by hetunandu — CHANGES_REQUESTED** [review]
Did not work for me. History was updated though. Why dont we implement the smooth scrolling already present

**Comment by rishabhsaxena** [comment]
@hetunandu found an issue, have updates the `ids` to use `slug` instead of `orgName`

File: app/client/src/sagas/OrgSagas.ts [hunk 1]
```diff
@@ -221,6 +222,10 @@ export function* createOrgSaga(
       });
       yield call(resolve);
     }
+
+    // get created org in focus
+    const slug = response.data.slug;
+    history.push(`${window.location.pathname}#${slug}`);
```
Comment (hetunandu) [inline app/client/src/sagas/OrgSagas.ts:228]: history.push is leading to a refresh.

**Review by hetunandu — CHANGES_REQUESTED** [review]

File: app/client/src/sagas/OrgSagas.ts (hunk 1 above)
Comment (rishabhsaxena) [reply app/client/src/sagas/OrgSagas.ts:228]: actually we're doing a `reload` within `switchUserOrgSaga`, not sure why it's needed there, may be we can fix it as a separate issue?

Comment (hetunandu) [reply app/client/src/sagas/OrgSagas.ts:228]: Lets do it with this PR itself

**Comment by rishabhsaxena** [comment]
@hetunandu have pushed the updates

**Review by hetunandu — APPROVED** [review]

**Pull request merged**